changelog.md

```diff
@@ -37,22 +37,44 @@ Health checks now work with HTTPS.
 
 #### Faster (configurable) task distribution
 
-In order to speed up task launching and use the resource offers Marathon receives from Mesos more efficiently,
-we added a new offer matching algorithm, that tries to start as many tasks as possible per task offer cycle.
-The maximum numbers of tasks to start are configurable with the follwing startup parameters:
+Mesos frequently sends resource offers to Marathon (and all other frameworks). Each offer will represent the available resources of a single node in the cluster. Until now, Marathon would only start a single task per resource offer, which led to slow task launching in smaller clusters. In order to speed up task launching and use the resource offers Marathon receives from Mesos more efficiently, we added a new offer matching algorithm, that tries to start as many tasks as possible per task offer cycle. The maximum numbers of tasks to start are configurable with the following startup parameters:
```

Comment:
Mesos frequently sends resource offers to Marathon (and all other frameworks). Each offer will represent the available resources of a single node in the cluster. Before this change, Marathon would only start a single task per resource offer, which led to slow task launching in smaller clusters. In order to speed up task launching and use the resource offers Marathon receives from Mesos more efficiently, we added a new offer matching algorithm which tries to start as many tasks as possible per task offer cycle. The maximum number of tasks to start is configurable with the following startup parameters: